packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +116,163 @@ class DefaultTaggedText extends StatelessWidget {
     );
   }
 }
+
+/// A widget that takes two children and lays them out along one axis.
```

Comment:
This summary is missing the key bit that the user resize the widgets along that axis.

Reply:
Good catch, done.